### ISSUE TITLE
Is there a reason youtube is in forbidden with "-f mp4"?

### ISSUE THREAD

**Issue by SnabbSpelare** [yt-dlp/yt-dlp#10506]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm asking a question and **not** reporting a bug or requesting a feature
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Please make sure the question is worded well enough to be understood

I'm having an issue where me using yt-dlp -f mp4 (link) causes it to tell me YouTube is forbidden. This worked previously.

See image blow.
![image](https://github.com/user-attachments/assets/30b5d5d8-42e1-4c49-a0eb-9cf8f385adde)

### Provide verbose output that clearly demonstrates the problem

- [ ] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [ ] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
C:\Users\Username>yt-dlp -f mp4 https://youtu.be/LlB7AdV8bw4
[youtube] Extracting URL: https://youtu.be/LlB7AdV8bw4
[youtube] LlB7AdV8bw4: Downloading webpage
[youtube] LlB7AdV8bw4: Downloading ios player API JSON
[youtube] LlB7AdV8bw4: Downloading player d60b0ef9
WARNING: [youtube] LlB7AdV8bw4: nsig extraction failed: You may experience throttling for some formats
         n = L_xFBAVqIoh4TVAh ; player = https://www.youtube.com/s/player/d60b0ef9/player_ias.vflset/en_US/base.js
WARNING: [youtube] LlB7AdV8bw4: nsig extraction failed: You may experience throttling for some formats
         n = FRWjHfdKHDqJj3DS ; player = https://www.youtube.com/s/player/d60b0ef9/player_ias.vflset/en_US/base.js
[youtube] LlB7AdV8bw4: Downloading m3u8 information
[info] LlB7AdV8bw4: Downloading 1 format(s): 18
ERROR: unable to download video data: HTTP Error 403: Forbidden
```

**Comment by seproDev** [comment]
Two things:
1. You are running an outdated version of yt-dlp. Update.
2. `-f mp4` should nearly never be used. It means to get the best **premerged** mp4 file, which in the case of youtube means 360p. If you want an mp4 file (with h264 and aac) use: `-S vcodec:avc,res,acodec:aac`. Make sure you have ffmpeg installed for this to work.

**Comment by SnabbSpelare** [comment]
Sorry really thought it was updated or automatically updated but I shouldn't trust that instict. But I have ffmpeg installed so that's weird. Also why wouldn't I use what has always worked and gotten me YouTube shorts and etc. I noticed that yt-dlp can't get or capture videos even with a batch script such as: `yt-dlp -i --no-warnings --ffmpeg-location "C:\.bin\ffmpeg.exe" --throttled-rate 200K --sponsorblock-remove all --add-metadata --embed-metadata --embed-thumbnail --embed-info-json --audio-quality 0 --sub-format best --sub-langs all -f best --recode mp4 --download-archive ".\Archive.txt" -a ".\Links.txt" -o "Videos (YT)\%%(title)s.%%(ext)s"`
